packages/mdc-chips/README.md

```diff
@@ -174,10 +198,10 @@ Mixin | Description
 
 > _NOTE_: `mdc-chip-set-spacing` also sets the amount of space between a chip and the edge of the set it's contained in.
 
-### `MDCChip` and `MDCChipSet`
+## `MDCChip` and `MDCChipSet` Properties and Methods
 
 The MDC Chips module is comprised of two JavaScript classes:
-* `MDCChip` defines the behavior of a single chip
+* `MDCChip` defines the behavior of a single chip.
```

Comment:
On the line above this:

* module -> package (arguably each of those classes is in a JS module)
* "is comprised of" -> "is composed of" (comprises works the opposite way)